### PR TITLE
[phc] Remove import org.apache.common

### PR DESCRIPTION
**Pull request by lsiepel**
- Remove dependency on apache.commons

Provided some tests for the padding, rest is untested, minor refactoring from only from code perspective

### CODE REVIEW

File: bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+package org.openhab.binding.phc.internal.util;
+
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link StringUtils} class defines some static string utility methods
+ *
+ * @author Leo Siepel - Initial contribution
+ */
+public class StringUtils {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java:22]: Can you please add `@NonNullByDefault`

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java:22]: Now I see after merging that it was also missing in the new classes you created for the sonos and upnpcontrol bindings.
If you could add them, that would be better.

Comment (lsiepel) [reply bundles/org.openhab.binding.phc/src/main/java/org/openhab/binding/phc/internal/util/StringUtils.java:22]: Yes, done. Will do the others too.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**